controllers/errors.controllers.js

```diff
@@ -0,0 +1,15 @@
+exports.handleCustomErrors = (err, req, res, next) => {
+	if (err.status) {
+		res.status(err.status).send({ message: err.message });
+	}
+	next(err);
+};
+
+exports.handle500Errors = (err, req, res, next) => {
+	res.status(500).send({ message: 'Internal Server Error' });
+	next(err);
```

Comment:
There's no point invoking next here as this error handling middleware for internal server errors should be the very last one.